annotation/tests/test_distribution.py

```diff
@@ -1516,3 +1518,155 @@ def test_prepare_response():
             )
             == expected_output
         )
+
+
+@pytest.fixture
+def users_for_test():
+    yield (
+        User(user_id=1, default_load=10, overall_load=10),
+        User(user_id=2, default_load=10, overall_load=10),
+        User(user_id=3, default_load=2, overall_load=2),
+    )
+
+
+@pytest.fixture
+def tasks_for_test():
+    yield (
+        {
+            "task_id": 0,
+            "file_id": 10,
+            "pages": [1, 2, 3],
+            "user_id": 1,
+        },
+        {
+            "task_id": 1,
+            "file_id": 10,
+            "pages": [4, 5, 6],
+            "user_id": 2,
+        },
+        {
+            "task_id": 2,
+            "file_id": 10,
+            "pages": [1, 2, 3, 4, 5, 6],
+            "user_id": 2,
+        },
+    )
+
+
+@pytest.mark.parametrize(
+    (
+        "annotators_ids",
+        "validators_ids",
+        "validation_type",
+        "validation_files_to_distribute_ids",
+        "extensive_coverage",
+        "pages_in_work_setup",
+        "pages_in_work_ids",
+        "tasks_ids",
+        "expected_tasks_ids",
```

Comment:
it is overwhelming, please check out these vids, they might help you
https://www.youtube.com/watch?v=QIUrd327tOQ
https://www.youtube.com/watch?v=aQH7hyJn-No